### ISSUE TITLE
Update/Install re-adds content items if they have been modified to EmbeddedResource

### ISSUE THREAD

**Issue by pms1969** [fsprojects/Paket#1584]
If you have a package with content, and you want that content then to become an embedded resource (you switch it in VS), a `paket update` or `paket install` will add the file a second time to the project (i.e. it appears twice in VS).

Just wondering if this is a bug, or if there is some `paket.dependencies` jigpokery I can do to stop this.

### Repro steps

As described above.

1. add a package that has content.
2. open solution in VS
3. change content file to EmbeddedResource in properties pane
4. run `paket update`

### Expected behavior

I'd expect there to be only 1 file, or there to be a way in `paket.dependencies` to ensure that the content is embeded, but given that you may only want to do this to certain files, maybe not adding it again would be preferable.

### Actual behavior

file appears twice in the project.

### Known workarounds

unknown

### Related information 

* Operating system = Windows
* Branch = Paket version 2.58.6.0
* .NET Runtime 4.5

**Comment by forki** [comment]
yeah I guess this is not supported yet. we would need a repro sample ad then we can try to make that work

**Comment by sergey-tihon** [comment]
Yes, it is not supported, would be nice to know package name and type of content.

Now packet identify build action based on content type and then add to the project.
If you change build action to `EmbeddedResource` on the next install\update packet cannot find file (that he added) and add it once again with default build action.

**Comment by pms1969** [comment]
Can't get you a repro right now, but will try to remember to do it over the weekend.

On a related note, this is very much like (https://github.com/fsprojects/Paket/issues/1555).  Given both those issues,is it perhaps better to not manage content files from paket, except to copy or not copy them to the solution, and add if not there?  Would solve both issues, but there may be a need/use case that I'm unaware of.
